integration/clean/README.md

```diff
@@ -7,12 +7,22 @@ The cleaner first searches for ami names (these are the ami created by the pipel
 
 Then checks to see if the creation date is greater than 60 days. (The aws sdk v2 gives creation date as a pointer to string. To convert to golang time we use the aws smithy go time. This allows us to compare to 60 days in past time)
 
-If the ami is older than 60 days old then we delete the ami
+If the ami is older than 30 days old as default (can be configured) then we delete the ami
```

Comment:
If we're going to indicate that this is configurable, we should say how it is configured.

Reply:
Good point to me again. Have changed the document how to configure this parameter